CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Changed
+- Added an `external_import` section to the `Bender.yml` to replace the `vendor.py` import with the new bender feature.
```

Comment:
Needs to be rebased on top of master since we moved to 0.3.4

Reply:
Done